gateway/src/service.rs

```diff
@@ -428,7 +428,18 @@ impl GatewayService {
             let project = row.get::<SqlxJson<Project>, _>("project_state").0;
             if project.is_destroyed() {
                 // But is in `::Destroyed` state, recreate it
-                let project = Project::create(project_name.clone());
```

Comment:
`Project::create` does not allow to customise the `ProjectCreating` payload, so manually creating the latter using the existing builder pattern. Perhaps there could be an ad-hoc builder for Project itself.